Daf.Meta/Model.cs

```diff
@@ -952,6 +1000,12 @@ public static void Deserialize(string folder)
 											hubRelationship.NotifyPropertyChanged("HubMapping");
 										};
 
+										hubMapping.ChangedStagingColumn += (s, e) =>
+										{
+											// I dislike having this method be internal instead of private. Need advice on how to do it better.
```

Comment:
Let's discuss it!

Reply:
I've been considering this and I've come up with a solution I believe to be better but it would require a bit of an overhaul. I think the current solution ought to be good enough for now. Let me know your thoughts.